src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java

```diff
@@ -298,6 +298,7 @@ public void loadNBTData(NBTTagCompound aNBT) {
             // backward compatibility
             voidingMode = aNBT.getBoolean(VOID_EXCESS_NBT_KEY) ? VoidingMode.VOID_ALL : VoidingMode.VOID_NONE;
         }
+        if (!getAllowedVoidingModes().contains(voidingMode)) voidingMode = getDefaultVoidingMode();
```

Comment:
nit: I prefer putting brackets
```suggestion
        if (!getAllowedVoidingModes().contains(voidingMode)) {
            voidingMode = getDefaultVoidingMode();
        }
```

Reply:
no braces seem to be the standard, e.g. L278

Reply:
ok